### DISCUSSION TITLE
[Feature request] Give access to information in AppStream metainfo of available version in UpdateMonitor portal

### DISCUSSION THREAD

**Discussion by Corristo** [flatpak/xdg-desktop-portal#1266] [Ideas]
Hello everyone,

I recently added self-update functionality to an application. The notification that a new version is available and the installation of this new version as well as restarting the application now all work fine, I only ran into one issue: I wanted to show the user not only that a version is available, but also the version number of that new version (so they can see whether it is a major or minor release), and ideally also the description of the release as in the AppStream metainfo xml file. 

As far as I can tell this information currently is not accessible, neither through the UpdateMonitor portal directly, nor through the Flatpak portal. So I had to resort to fetching the additional information from the metainfo.xml file hosted on a dedicated web space, which then requires to allow network access in the sandbox even if it otherwise wouldn't be needed. 

Since multiple software managers (I've tested Gnome Software and Plasma Discover) are able to show the version and the description of the release before installing it Flatpak must already have some way to access it, it just isn't exposed through the portals. It would be nice to get access to that information through either the UpdateMonitor  (only for the remote version) or the Flatpak portal (to allow access to the metainfo for any given commit).

**Comment by TingPing** [comment]
I think its fine to discuss here, but worth noting `UpdateMonitor` is a part of `flatpak` and not `xdg-desktop-portal`.

**Reply by Corristo** [reply]
Oh, I wasn't aware there are portals that are not part of xdg-desktop-portal. I've created a feature request in the `Flatpak` repo now (https://github.com/flatpak/flatpak/issues/5661), so I'll close this discussion.
